pkg/armhelpers/azureclient.go

```diff
@@ -80,6 +80,11 @@ type AzureClient struct {
 	servicePrincipalsClient graphrbac.ServicePrincipalsClient
 }
 
+// GetKubernetesClient returns a KubernetesClient hooked up to the api server at the apiserverURL.
```

Comment:
Thanks for moving this `AzureClient` pointer method here alongside its other methods. (Unfortunately this should never have been implemented as a method, as it neither operates upon nor modifies the instance object. :/ (But we don't want to worry about that now)